### DISCUSSION TITLE
Extraction of triangle symbols

### DISCUSSION THREAD

**Discussion by meghanaviyyapu** [pymupdf/PyMuPDF#3130] [Looking for help]
I have tried extracting symbols like 
![image](https://github.com/pymupdf/PyMuPDF/assets/36301306/43fc4df0-a355-4155-97d7-b2e10f7bcbb9)
from PDF. The symbol is getting extracted as u when I print span["text"]. Could you let me know how it can be handled?

**Comment by meghanaviyyapu** [comment]
https://web.prudential.com/media/managed/GI_BTA_Flyer.pdf

This is the PDF. I am trying to extract the triangle symbols but the extracted text is "u"

**Reply by JorjMcKie** [reply]
The visual appearance is fully under control of the font. You cannot do anything about this and you **_never_** possibly can know how extracted text originally looked like.
The PDF creator in this case unfortunately has decided to not use any of the symbols fonts.

**Reply by JorjMcKie** [reply]
BTW all other text extraction tools also return a "u".

**Reply by meghanaviyyapu** [reply]
Oh is there no way to know how the original text looks like? Is contacting the creator of the PDF the only solution?

**Reply by JorjMcKie** [reply]
> Oh is there no way to know how the original text looks like? Is contacting the creator of the PDF the only solution?

Unfortunately no other solution. Yes, you may want to contact the creator and shout at him.
Predicting his answer: "Who gave you the permission to extract our text?!" 😉

**Reply by meghanaviyyapu** [reply]
Thanks for clarifying
